### PR TITLE
Update CSP billing adapter namespace

### PR DESCRIPTION
**Pull request by guangyee**
To support PAYG offering, we need to have an umbrella helm chart to encapsulate Rancher, usage operator, and billing adapter into a single release. Since Rancher must be using the cattle-system namespace, this will restrict the umbrella chart to confine to the same namespace. Furthermore, helm does not effectively support installing subcharts into different namespaces without hardcoding it into the subcharts, we must also need to check for the PAYG CSP billing adapter in the same namespace as Rancher.

## Issue: 
The original PR #41849 assumed that Rancher can be packaged as a subchart in the PAYG marketplace offerring. Unfortunately, Rancher chart must be installed into `cattle-system` namespace instead of `cattle-csp-billing-adapter-system` namespace as the original PR intended.
 
## Problem
The original PR #41849 assumed that Rancher can be packaged as a subchart in the PAYG marketplace offerring. Unfortunately, Rancher chart must be installed into `cattle-system` namespace instead of `cattle-csp-billing-adapter-system` namespace as the original PR intended.
 
## Solution
In order to successfully package Rancher into an umbrella chart, we must also move the PAYG CSP billing adapter to
`cattle-system` as well.
 
## Testing
The original test plan still applicable.

### Manual Testing
1. Manually built the Rancher image with the changes.
2. Ensure support config is successfully generated.

### Automated Testing

* Test types added/modified:
    * Unit
    * Integration (Go Framework)
    * Integration (v2prov Framework)
    * Validation (Go Framework)
    * Other - Explain: _EXPLAIN_
    * None
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
* If "None" - Reason: _EXPLAIN THE REASON_
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

### Regressions Considerations

_TODO_

Existing / newly added automated tests that provide evidence there are no regressions:
* Existing unit tests applicable to this change.

### CODE REVIEW

File: pkg/managedcharts/cspadapter/cspadapter.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ const (
 	MLOChartName = "rancher-csp-adapter"
 	// PAYGChartNamespace is the namespace that we expect the adapter to be installed in,
 	// for the new Pay-As-You-Go (PAYG) license offering
-	PAYGChartNamespace = "cattle-csp-billing-adapter-system"
+	PAYGChartNamespace = "cattle-system"
```
Comment (cbron) [inline pkg/managedcharts/cspadapter/cspadapter.go:22]: If it's going into cattle-system, you'll also need to add this to the system project here: https://github.com/rancher/rancher/blob/release/v2.7/pkg/settings/setting.go#L27

That also begs question, how does it impact backup/restore operator ? CSP-adapter is excluded, so does this follow same model ? https://github.com/rancher/backup-restore-operator/blob/914ffa31809250e2dda847625dca8389418ffbeb/charts/rancher-backup/files/default-resourceset-contents/rancher.yaml#L14

**Comment by jeremy-moffitt** [comment]
filed https://github.com/rancher/rancher/issues/42347  to track this... we will ideally want this into the Q3 milestone ( https://github.com/rancher/rancher/milestone/313 )

File: pkg/managedcharts/cspadapter/cspadapter.go (hunk 1 above)
Comment (guangyee) [reply pkg/managedcharts/cspadapter/cspadapter.go:22]: Can you please elaborate? Per my understanding, the new PAYG CSP billing adapter will be maintained outside of Rancher. But I am not aware of addition requirements for putting it under the cattle-system namespace.

**Comment by jeremy-moffitt** [comment]
on hold while we explore alternative solutions to the problem

**Comment by guangyee** [comment]
No longer needed. The deployer container approach is a viable solution.

**Pull request closed without merging**